core/src/main/scala/tech/beshu/ror/accesscontrol/factory/decoders/rules/auth/LdapRulesDecoders.scala

```diff
@@ -221,6 +255,10 @@ private object LdapRulesDecodersHelper {
     s"${ruleName.show} rule requires to define 'groups_or'/'groups' or 'groups_and' arrays (but not both)"
   }
 
+  private[rules] def negativeGroupsRuleNotAllowedForLdapWithGroupsFiltering = {
+    s"It is not allowed to use groups_not_any_of and groups_not_all_of rule, when LDAP server-side groups filtering is enabled. Consider using a combined rule, which applies two rules at the same time: groups_or/groups_and together with groups_not_any_of/groups_not_all_of."
```

Comment:
Do we support the combined rule in this change? 
If not, let's change the message for now.

Reply:
The combined rule is supported for LDAP. Just without the new config syntax (making a config section for both lists), which will be added in RORDEV-1388.